packages/plugin-vue/src/index.ts

```diff
@@ -122,9 +122,6 @@ export default function vuePlugin(rawOptions: Options = {}): Plugin {
         define: {
           __VUE_OPTIONS_API__: config.define?.__VUE_OPTIONS_API__ ?? true,
           __VUE_PROD_DEVTOOLS__: config.define?.__VUE_PROD_DEVTOOLS__ ?? false
-        },
-        ssr: {
-          external: ['vue', '@vue/server-renderer']
```

Comment:
I think we should still keep these if `legacy.buildSsrCjsExternalHeuristics` is defined

Reply:
Good catch! It was in my memory until editing the code 😅